index.js

```diff
@@ -177,7 +178,7 @@ export default class QRCodeScanner extends Component {
       } else {
         return (
           <View style={styles.rectangleContainer}>
-            <View style={styles.rectangle} />
+            <View style={[styles.rectangle, this.props.markerStyle]} />
```

Comment:
do we have a default prop for markerStyle?
otherwise it'll throw an error for existing usage of the module where this prop isn't passed. 

alternatively, we could add a check for existence of the prop before passing it into the array like so:
`<View style={[styles.rectangle, this.props.markerStyle ? this.props.markerStyle : null]} />`

Reply:
> default prop for markerStyle

I added a default prop for `markerStyle` on line 35.

Reply:
Ohh, I see what you're saying...